site/content/posts/announcing-kudo-0.3.0.md

```diff
@@ -0,0 +1,69 @@
+
+## Release Highlights
+
+### Switch from Framework to Operator
+Framework was the term the project previously used to represent a service that KUDO "the operator" would be instrumented to be an operator for. The intention was to bring clarity by introducing a new term. It was thought that the new term would allow for more accuracy in technical communication by reduce the overloading of the term operator. For some it caused more confusion and almost always resulted in the need for explanation. We have abandon the term completely resulting in a **breaking** change to the CRDs.
+
+All previous concepts and required objects using the word Framework have been replaced with Operator, resulting Operator and OperatorVersion in the type definition.
+
+This will result in some overloading of the term operator, but aligns better with the eco-system. The running KUDO controller is an operator that manages operators built with KUDO's packaging format. In the future, this will be expanded with support for Helm charts (v0.4.0), other packaging formats, existing operators, and beyond. KUDO's mission is to enable easy, declarative day two operations and API management for complex applications, no matter how they were originally built.
+
+### Package Format
+
+[KEP-10](https://github.com/kudobuilder/kudo/blob/v0.3.0/keps/0010-package-manager.md) for the package manager is underway, with release providing 2 features; package repository and a package format.
+
+The general format allows for a number of files, but starts with an `operator.yaml`.  A common layout might be:
+
+```
+operator.yaml
+params.yaml
+templates/
+     <additional files>.yaml
+```
+The `operator.yaml` defines the operator and its plans.  `params.yaml` defines the params to be used through all templated files.  While the templated files are defined under the templates folder. The templated files are referenced in the `operator.yaml` file.  The zookeeper operator provides a [good example](https://github.com/kudobuilder/operators/tree/a1f4cf/repository/zookeeper/0.1.0).
+
+
+These package files are group together in a folder or subfolder or can be tarred together in a `tgz` file providing a couple of ways to install an operator.
+
+### Package Installation Options
+
+Introduced with this released are 2 ways to install an operator to KUDO. The first is designed for operator developers which is to install from the local filesystem. This can be either a folder with the operator package files or a tarball of those files.
```

Comment:
release

Reply:
```suggestion
Introduced with this release are 2 ways to install an operator to KUDO. The first is designed for operator developers which is to install from the local filesystem. This can be either a folder with the operator package files or a tarball of those files.
```